common/lib/xmodule/xmodule/video_module/video_module.py

```diff
@@ -603,8 +603,10 @@ def editable_metadata_fields(self):
 
         languages = [{'label': label, 'code': lang} for lang, label in settings.ALL_LANGUAGES]
         languages.sort(key=lambda l: l['label'])
+        editable_fields['transcripts']['custom'] = True
         editable_fields['transcripts']['languages'] = languages
         editable_fields['transcripts']['type'] = 'VideoTranslations'
+        editable_fields['transcripts']['value'] = {}
```

Comment:
We can pass the actual transcripts data here. This way we don't have to do any of the below changes in this file and also we don't need do any changes in `cms/templates/widgets/video/transcripts.html` and `cms/static/js/views/video/transcripts/editor.js`